content/en/cluster.md

```diff
@@ -2,26 +2,22 @@
 title: Cluster
 status: Completed
 category: Concept
-tags: ["infrastructure", "fundamental", ""]
+tags: ["infrastructure", "fundamental", "architecture"]
 ---
 
-A cluster is a group of computers or applications that work together towards a common goal. 
-In the context of cloud native computing, the term is most often applied to [Kubernetes](/kubernetes/). 
-A Kubernetes cluster is a set of services (or workloads) that run in their own containers, usually on different machines. 
-The collection of all these [containerized](/containerization/) services, connected over a network, represent a cluster.
+A cluster is a group of computers or applications called [nodes](/nodes/) that work together towards a common goal.
```

Comment:
Do we want to clarify this further that this could be actual hardware or virtual hardware?

Reply:
My personal opinion is that this nuance is not really important in order to understand the term. Also when following the link to the node term it is explained there that it can be virtually or physically. I will go on and merge, we can still open a new PR if we feel the need to be more clear.